package.json

```diff
@@ -118,11 +118,16 @@
   },
   "lint-staged": {
     "linters": {
-      "*.{ts,tsx}": [
+      "modules/**/*.{ts,tsx}": [
```

Comment:
This prevents a TSLint warning about the file passed in is excluded by the `tsconfig.json` configuration settings. The project has 2 different Typescript projects - `modules` and `cypress`.

* `modules` - All modules source code goes here and includes the `jest` Typescript namespace and config to create a Storybook application and run Jest tests against glob patterns
* `cypress` - All source code here includes the `cypress` Typescript namespace because it conflicts with the `jest` namespace and requires a different config (mostly shared). The lint-staged needs to reference these different configs to be processed correctly